app/forms/profile_form.rb

```diff
@@ -0,0 +1,35 @@
+class ProfileForm
```

Comment:
Given what I said above about extracting the resume to it's own service object, I don't think we really need this. Most the complications came because of that, and everything else is just simple rails form stuff which should be in the model.

Reply:
How do you typically approach handling form data when it's not associated with a particular resource on the unrelated resource's controller? Do you namespace it differently in the form? Do you create a separate `params` method that requires those data, e.g. 
```ruby
def resume_params
  params.require(:profile).permit(:resume, :resume_name, :current_resume_id)
end
```

Reply:
I don't have too much of an opinion. separate params would be totally fine by me though

Reply:
@toyhammered -- since there's no attribute for resumes on the `Profile` model, it becomes problematic to leverage rails form helpers. The `ProfileForm` addresses this issue.
I can still update so that the command is only updating the resume, but I'm going to push back on removing the Form Object here.